.github/ISSUE_TEMPLATES/Feature Request.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Feature Request
+about: Suggest an idea for this project
+title: "[FEATURE]"
+labels: ''
+assignees: ''
+
+---
+
+## Feature Request
+
+- [ ] Related to an existing [Issue](../../../issues) 
```

Comment:
```suggestion
- [ ] Related to an existing [Issue](../issues) 
```

Reply:
I think both paths were wrong. Added the correct one.